src/bots/deepseek/DeepSeekAPICoderBot.js

```
@@ -0,0 +1,10 @@
+import DeepSeekAPIBot from "./DeepSeekAPIBot";
+
+export default class DeepSeekAPICoderBot extends DeepSeekAPIBot {
+  static _className = "DeepSeekAPICoderBot";
+  static _model = "deepseek-coder";
+
+  constructor() {
+    super();
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)